src/tests/nativeaot/SmokeTests/StackTraceMetadata/BodyFoldingTest.cs

```diff
@@ -3,9 +3,16 @@
 
 using System;
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 
 class BodyFoldingTest
 {
+    [UnmanagedCallersOnly(EntryPoint = "FoldableMethod1")]
+    static void FoldableMethod1() { }
+
+    [UnmanagedCallersOnly(EntryPoint = "FoldableMethod2")]
+    static void FoldableMethod2() { }
```

Comment:
Shouldn't it (all `UnmanagedCallersOnly(EntryPoint`-decorated methods) be exported by default or do we need `--export-unmanaged-entrypoints` for this method to be exported?

Reply:
They are not exported if the output is an EXE. This is an EXE.